db/schema.rb

```diff
@@ -761,7 +761,15 @@
     t.boolean  "immigrant"
     t.datetime "uwfs_training_date"
     t.boolean  "clearinghouse_record_found"
+    t.boolean  "international_student"
+    t.string   "intended_major"
+    t.integer  "transfer_student"
+    t.integer  "residency_status"
+    t.string   "home_state"
+    t.integer  "housing_status"
     t.boolean  "parental_consent_on_file"
+    t.boolean  "pending_participant"
+    t.string   "intake_form_signate"
```

Comment:
I think this means that you have the fields in your local sqlite database. The `schema.rb` is generated after each migration based on the current database structure, I believe. So I think you have to manually remove those fields in the sqlite console. Unfortunately sqlite3 doesn't support simply deleting a column; you have to recreate the whole table. One option would be to delete your sqlite3 file and start over, or you can do the slightly complex table recreate: http://stackoverflow.com/questions/5938048/delete-column-from-sql-table/5987838#5987838